tests/unit/amo/helpers.js

```diff
@@ -218,11 +218,19 @@ export function dispatchClientMetadata({
 
 export function dispatchSignInActions({
   authToken = userAuthToken(),
+  biography = 'I love making add-ons!',
```

Comment:
Instead of duplicating all these defaults what about accepting a `user` parameter to `dispatchSignInActions()` that overrides `loadCurrentUserAccount({ user })`? You could keep the existing shortcuts used by existing tests. With that change you can make use of the `createUserAccountResponse()` helper.

Reply:
Oh, sweet, I brute-forced this without really looking, but that's much better. Will do!

Reply:
For what it's worth it wasn't much effort to remove the existing shortcuts and I think it makes it more consistent and removes the temptation/obviousness of adding more shortcuts in the future.